app/main/views/drafts.py

```diff
@@ -347,6 +348,44 @@ def complete_draft_service(draft_id):
     return jsonify(services=draft.serialize()), 200
 
 
+@main.route('/draft-services/<int:draft_id>/update-status', methods=['POST'])
+def update_draft_service_status(draft_id):
+    updater_json = validate_and_return_updater_request()
+    update_json = validate_and_return_draft_request()
+    json_only_has_required_keys(update_json, ['status'])
+
+    new_status = update_json['status']
+    if not new_status or new_status not in DraftService.STATUSES:
```

Comment:
I don't think we need to do this since status validator on `ServiceTableMixin` would run the same check